.github/workflows/release.yml

```
@@ -1,60 +1,33 @@
-  GH_USER_NAME: github.actor
-  SCRIPTS_VERSION: 5.8.0
-  REPOSITORY_URL: 'https://maven.pkg.github.com/'
+      - '.github/workflows/chart-release.yml'
 
 jobs:
   release:
     runs-on: ubuntu-latest
+
     steps:
       - name: Checkout
         uses: actions/checkout@v3
```

actionlint (1.7.11)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)